frontend/shared/components/src/events/EditEventView.vue

```diff
@@ -310,21 +311,36 @@ const organization = useOrganization();
 const present = usePresent();
 const platform = usePlatform();
 
-const {externalOrganization, choose: chooseOrganizer, loading, errorBox: organizerEventBox} = useExternalOrganization(
-    computed({
-        get: () => patched.value.organizationId,
-        set: (organizationId) => addPatch({
-            organizationId
-        })
-    })
-)
+useValidation(errors.validator, () => {
+    const se = new SimpleErrors()
+
+    if(isLocationRequired.value) {
+        if(!location.value) {
+            se.addError(new SimpleError({
+                code: "invalid_field",
+                message: "De locatie is verplicht voor deze soort activiteit.",
+                field: "event_required"
+            }))
+        }
+    }
+
+    if (se.errors.length > 0) {
+        errors.errorBox = new ErrorBox(se)
+        return false
+    }
 
+    errors.errorBox = null
```

Comment:
Dit is een klein beetje gevaarlijk omdat dit niet meer gaat werken als je later meerdere `useValidation` zou toevoegen in dezelfde component. Dus ik zou het op een andere manier aanpakken, want dit is niet zo een mooi patroon.

Reply:
(het probleem is dus dat je hier met een gemeenschappelijke errorBox werkt, terwijl `useValidation` normaal enkel is voor kleine componenten waar je zeker bent dat je maar één keer `useValidation` moet gebruiken)

Reply:
Ik zou het hier dus gewoon verplaatsen naar `async function save() {` of een custom save method die je daar aanroept, en waarin je gewoon throwt in plaats van errorBox aan te passen